pax/config/Simulation.ini

```diff
@@ -5,3 +5,1571 @@ look_for_config_in_runs_db = False
+        },
+    87:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    88:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    89:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    90:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    91:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    92:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    93:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    94:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    95:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    96:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    97:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    98:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    99:{
+        'CH4+':  {'p': 2. * 1.8/4.9,
+                  'time_distribution': 'normal',
+                  'time_parameters': dict(loc=1 * us,
+                                        scale=10. * ns)},
+        },
+    100:{
+        'H2':  {'p': 0.002 * 1.8/4.9,
```

Comment:
That's only for temporary test. A complete one will replace this once we get the inputs from PMT after pulse analysis.